### PR TITLE
Feature/partial readmes

### PR DESCRIPTION
**Pull request by okhan-okbay-cko**
Our README was colossal. So, this created partial/sub READMEs

Preview available here: https://github.com/checkout/frames-ios/blob/1e35bf770c058c36c03f09e18451df051209db4a/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,23 +17,21 @@
...
+- [Get started](https://github.com/checkout/frames-ios/blob/feature/partial-readmes/.github/Sub-READMEs/GetStarted.md): _Start testing what you can achieve by presenting inside your Applications UI_ 
 
-- [Make it your own](#Make-it-your-own): _Customising the UI to become a part of your app_
+- [Make it your own](https://github.com/checkout/frames-ios/blob/feature/partial-readmes/.github/Sub-READMEs/MakeItYourOwn.md): _Customising the UI to become a part of your app_
 
-- [Other features](#Other-features): _How we help with Apple Pay & 3D Secure Challenges_
+- [Other features](https://github.com/checkout/frames-ios/blob/feature/partial-readmes/.github/Sub-READMEs/OtherFeatures.md): _How we help with Apple Pay & 3D Secure Challenges_
 
-- [Migrating](#Migrating-from-3.X.X-to-4.X.X): _If you have used 3.5.x version (or lower) before_
+- [Migrating](https://github.com/checkout/frames-ios/blob/feature/partial-readmes/.github/Sub-READMEs/Migration.md): _If you have used 3.5.x version (or lower) before_
```
Comment (okhan-okbay-cko) [inline README.md:34]: refs willbe updated to main branch before merging

File: .github/Sub-READMEs/Migration.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Migrating from 3.X.X to 4.X.X 
+3DS and Apple Pay processing remain unaffected so using them should still work the same. 
+
+If you're using Frames iOS from versions prior to v4 (we would strongly recommend using the latest release), this update will bring breaking changes that'll require a little development time.
+
+Because of our efforts to greatly improve the UI of the Frames and enabling you to customise it to such extents, the approach required is very different. This will require you to:
+- remove usage of Frames older version from your code base. This may be an opportunity to remove a screen as well, or a few other objects that were created only to support older Frames integration!
+- [Get started](https://github.com/checkout/frames-ios/blob/feature/partial-readmes/.github/Sub-READMEs/GetStarted.md)
```
Comment (okhan-okbay-cko) [inline .github/Sub-READMEs/Migration.md:8]: ref will be updated to main branch before merging

File: .github/Sub-READMEs/MakeItYourOwn.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
+# Make it your own
+<sub> Any customisation needs to be done before creating the ViewController. Once style is submitted to the factory, any changes done on it will not be reflected in the UI </sub>
+
+:exclamation:**Note that building your own UI can place you in a higher band of PCI compliance. In order to remain in the lowest level of PCI compliance, we recommend using one of the UI options listed below**:exclamation:
+ 
+In order of complexity we'll start with:
+
+
+### Modify Default
+In our [Get started](#Get-started) example we have used Default Style to get something working quickly. If that was mostly what you were looking for, then you'll be happy to know that each component is mutable and you should easily be able to customise individual properties. This is also used as example in our demo projects inside `Factory.swift`, in the method `getDefaultPaymentViewController`.
```
Comment (jheng-hao-lin-cko) [inline .github/Sub-READMEs/MakeItYourOwn.md:10]: The `Get started` link to be updated

**Review by jheng-hao-lin-cko — COMMENTED** [review]
Love the new format!

File: .github/Sub-READMEs/MakeItYourOwn.md (hunk 3 above)
Comment (okhan-okbay-cko) [reply .github/Sub-READMEs/MakeItYourOwn.md]: good catch! updated

File: .github/partial-readmes/MakeItYourOwn.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
+# Make it your own
+<sub> Any customisation needs to be done before creating the ViewController. Once style is submitted to the factory, any changes done on it will not be reflected in the UI </sub>
+
+:exclamation:**Note that building your own UI can place you in a higher band of PCI compliance. In order to remain in the lowest level of PCI compliance, we recommend using one of the UI options listed below**:exclamation:
+ 
+In order of complexity we'll start with:
+
+
+### Modify Default
+In our [Get started](https://github.com/checkout/frames-ios/blob/feature/partial-readmes/.github/partial-readmes/GetStarted.md) example we have used Default Style to get something working quickly. If that was mostly what you were looking for, then you'll be happy to know that each component is mutable and you should easily be able to customise individual properties. This is also used as example in our demo projects inside `Factory.swift`, in the method `getDefaultPaymentViewController`.
```
Comment (okhan-okbay-cko) [inline .github/partial-readmes/MakeItYourOwn.md:10]: update with main branch ref before merging

**Review by patrick-hoban-cko — APPROVED** [review]

**Review by jheng-hao-lin-cko — APPROVED** [review]

**Pull request merged**